packages/loader/driver-utils/src/test/summaryCompresssionTester.spec.ts

```diff
@@ -616,15 +623,15 @@ function checkCompressionConfig(
 	);
 }
 
-function getHeaderContent(summary: ISummaryTree) {
+function getHeaderContent(summary: ISummaryTree): any {
 	return getHeader(summary)["content"];
 }
 
-function getHeader(summary: ISummaryTree) {
+function getHeader(summary: ISummaryTree): SummaryObject {
```

Comment:
above we read the "content" member off of this, so maybe its supposed to return `ISummaryBlob`?